extensions/amp-ad-network-adsense-impl/0.1/responsive-state.js

```diff
@@ -149,6 +184,32 @@ export class ResponsiveState {
     return /** @type {!ResponsiveState} */ (state);
   }
 
+  /**
+   * Convert the element to container width responsive.
+   *
+   * @param {!Element} element
+   * @return {!Promise<?ResponsiveState>} a promise that return container width responsive state.
+   * @private
+   */
+  static convertToContainerWidth_(element) {
+    const vsync = Services.vsyncFor(toWin(element.ownerDocument.defaultView));
+
+    return vsync
+      .mutatePromise(() => {
+        const width = String(element./*OK*/ parentElement./*OK*/ clientWidth);
```

Comment:
There seems to be a bit of confusion. This measure needs to be done inside a `measurePromise`. The mutates directly after need to be done inside a `mutatePromise`.